sdk/communication/AzureCommunicationChat/Source/AzureCommunicationChatClient.swift

```diff
@@ -116,7 +116,7 @@ public final class AzureCommunicationChatClient: PipelineClient, PageableClient
         self.request(request, context: context) { result, httpResponse in
             let dispatchQueue = options?.dispatchQueue ?? self.commonOptions.dispatchQueue ?? DispatchQueue.main
             guard let data = httpResponse?.data else {
-                let noDataError = AzureError.sdk("Response data expected but not found.")
+                let noDataError = AzureError.client("Response data expected but not found.")
```

Comment:
Since the changes under sdk/communication are generated, should we fix the Autorest.swift code generator first and then republished the generated code here?